### PR TITLE
yurthub verify bootstrap ca on start

### PR DESCRIPTION
**Pull request by gnunu**
/kind bug

#### What this PR does / why we need it:
verify existed ca file if any, and do cleanup if necessary (mismatch). This will fix the openyurt convert failure due to stale contents left over in the yurthub root dir (/var/lib/yurthub).

#### Which issue(s) this PR fixes:

Fixes #

#### Special notes for your reviewer:

/assign @rambohe-ch 

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: pkg/yurthub/certificate/hubself/cert_mgr.go [hunk 1]
```diff
@@ -258,6 +274,24 @@ func (ycm *yurtHubCertManager) initCaCert() error {
...
 	}
 
+	if caExisted {
+		var existedCABytes []byte
+		var err error
+		existedCABytes, err = ioutil.ReadFile(caFile)
+		if err != nil {
+			klog.Infof("could not read existed %s ca cert, %v", caFile, err)
+		}
+		if (bytes.Compare(clusterCABytes, existedCABytes) == 0) {
```
Comment (rambohe-ch) [inline pkg/yurthub/certificate/hubself/cert_mgr.go:284]: when yurthub startup if cloud-edge network disconnected, so yurthub can not get `clusterCABytes` from kube-apiserver, leading to yurthub startup failure. so we can not use `CABytes` to verify whether remove certificates or not, and maybe we need to use `bootstrap token` from `yurthub parameter --join-token` and `/var/lib/yurthub/bootstrap-hub.conf` to verify.

Comment (gnunu) [reply pkg/yurthub/certificate/hubself/cert_mgr.go:284]: ok. Let me do bootstrap token way.

**Comment by rambohe-ch** [comment]
@gnunu A good job to remove pki based on verifying ca file. please fix the above golangci lint error.

**Comment by gnunu** [comment]
@rambohe-ch thanks!

**Comment by rambohe-ch** [comment]
@gnunu I think that maybe we can compare the remote server address from yurthub startup parameter `--serveraddr` and server address of `/var/lib/yurthub/bootstrap-hub.conf`. if yurthub is used to connect to a new kube-apiserver, the remote server address will be changed, so the old certificates should be removed.

**Review by rambohe-ch — APPROVED** [review]

**Pull request merged**